### DISCUSSION TITLE
How to set define plugin option of entry output filename

### DISCUSSION THREAD

**Discussion by Smanst3r** [webpack/webpack#17087] [Q&A] [answered]
Hi!
I'm looking for a webpack configuration option or a workaround to get the output file path
To set it further in the **definePlugin**
*****************************************************************************
Below is part of my configuration I believe may help:
```
const outputPath = APP_ENV === 'production'
    ? '<prod_path>/static/compiled'
    : <dev_path>/static/compiled'
const outputFileName = APP_ENV === 'production'
   ? '[name].[contenthash:8].js'
   : '[name].js'

{
...
    entry: {
        'app/layout': '<absolute_path_to_entry>/assets/js/layout.js', // layout.css is imported into the layout.js file
        ...
    },
    output: {
        path: outputPath
        filename: outputFileName,
        ...
    }
...
}
```

Result of webpack build in my outputPath:
IN development: *outputPath/app/layout.css*
IN production: *outputPath/app/layout.f45ca985.css* 

How do I get this path above to set the definePlugin option?
`
new webpack.DefinePlugin({
    APP_LAYOUT_CSS_PATH: <layout.css - absolute output file path>
})
`
*****************************************************************************
**PS:**
I need this *APP_LAYOUT_CSS_PATH* option inside my _ckeditor4/default-config.js_ file
to build ckeditor config `config.contentsCss = [APP_LAYOUT_CSS_PATH, ]`

Kind regards

**Comment by alexander-akait** [comment]
It is impossible, you can create manifest file from stats, save it as json, load it using `fetch` and using any infromation from manifest, there are a lot of plugins for this

**Reply by Smanst3r** [reply, marked as answer]
Thanks for reply @alexander-akait . 
I used to do this via XMLHttpRequest. 
I found a better solution for myself using the copy webpack plugin. Just copied source file to output path
